starter-core/src/main/java/io/micronaut/starter/build/gradle/GradleBuild.java

```diff
@@ -81,6 +77,11 @@ public List<GradlePlugin> getPlugins() {
         return plugins.stream().filter(gradlePlugin -> gradlePlugin.getGradleFile() == GradleFile.BUILD).collect(Collectors.toList());
     }
 
+    @NonNull
+    public List<String> getSettingsImports() {
+        return plugins.stream().filter(gradlePlugin -> gradlePlugin.getGradleFile() == GradleFile.SETTINGS).map(GradlePlugin::getSettingsImports).flatMap(Collection::stream).collect(Collectors.toList());
```

Comment:
```suggestion
        return plugins.stream().filter(gradlePlugin -> gradlePlugin.getGradleFile() == GradleFile.SETTINGS).map(GradlePlugin::getSettingsImports).flatMap(Collection::stream).toList();
```

Can use Java 17 `Stream.toList()`